telemetryproxy/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apk add --update make git
 ADD ./src /src
 WORKDIR /src
 
-RUN make VERSION=${VERSION} OTELCONTRIBCOL_FILENAME=otelcontribcol install-tools otelcontribcolbuilder
```

Comment:
`otelcontribcolbuilder` already depends on `install-tools` in the makefile, so this is redundant